Bukkit/src/main/java/com/plotsquared/bukkit/listener/PlayerEventListener.java

```diff
@@ -1009,17 +1011,11 @@ public void onInteract(PlayerInteractEvent event) {
                     return;
                 }
                 switch (type) {
-                    case ACACIA_BOAT:
-                    case BIRCH_BOAT:
                     case CHEST_MINECART:
                     case COMMAND_BLOCK_MINECART:
-                    case DARK_OAK_BOAT:
                     case FURNACE_MINECART:
                     case HOPPER_MINECART:
-                    case JUNGLE_BOAT:
                     case MINECART:
-                    case OAK_BOAT:
-                    case SPRUCE_BOAT:
```

Comment:
According to [its previus documentation](https://docs.improbable-game.fr/spigot-api/1.14.2/org/bukkit/event/entity/EntityPlaceEvent.html) this event wasn't triggered for boats pre 1.15. I'd prefer to keep this part, as while it doesn't resolve the bug on 1.13 and 1.14, it will keep functionality the same on those version.

In general, I'd prefer to handle all entity placements that can be handled that way using that event, as it's safer and cleaner.